server.js

```diff
@@ -3496,6 +3497,134 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub issue detail integration.
+camp.route(/^\/github\/(?:issues|pulls)\/detail\/(s|title|u|label|comments|age|last-update)\/([^\/]+)\/([^\/]+)\/(\d+)\.(svg|png|gif|jpg|json)$/,
+cache((queryParams, match, sendBadge, request) => {
+  const stateColor = s => ({ open: '2cbe4e', closed: 'cb2431', merged: '6f42c1' }[s]);
```

Comment:
This is going to re-define the function ever time this route is called, which is not really necessary as it's entirely static. Instead, pull this function out and make it a regular function in this file (or in one of the utility modules).

Reply:
Updated in 448e88c.